ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+// import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    backgroundColor: '#FAFAFA',
+    height: 200
+  },
+  centered: {
+    width: '100%'
+  },
+  title: {
+    fontSize: 14
+  }
+
+}))
+
+function FeatureCard (props: FeatureCardProps): JSX.Element {
+  const classes = useStyles()
+
+  return (
+    <Card className={`${classes.root}`} variant="outlined">
+        <CardContent>
+            <Grid container>
+                <Grid item xs={12}>
+                    <div className={classes.centered}>
+                        {props.icon}
+                    </div>
+                    <div className={classes.centered}>
+                        <Typography className={classes.title} color="textPrimary" gutterBottom>
+                        {props.title}
+                        </Typography>
+                        <Typography variant="body2" component="p" color="textSecondary" >
+                        {props.description}
+                        </Typography>
+                    </div>
+                </Grid>
+            </Grid>
+        </CardContent>
+    </Card>
+  )
+}
+
+interface FeatureCardProps {
+  id: string
```

Comment:
Is there a reason it should be a number rather than a string?  I thought about that but it seemed like a string would have an more implicit meaning.  I guess it takes up a few more bytes?

Reply:
This is probably open to discussion. Efficiency probably doesn't matter here as much. In my understanding, semantic meaning in a key is actually not desirable, because if it changes then the key might change.